projects/fine_tuning/visualizations/fine_tuning/store/parsers.py

```diff
@@ -402,16 +402,44 @@ def _parse_workload_config(dirname, locations):
 @helpers_store_parsers.ignore_file_not_found
 def _parse_ray_logs(dirname):
     ray_metrics = types.SimpleNamespace()
-    ray_metrics.summary = types.SimpleNamespace()
-    ray_metrics.summary.time = None
 
+    ray_metrics.summary = types.SimpleNamespace()
     ray_metrics.progress = []
 
     with open(register_important_file(dirname, artifact_paths.FINE_TUNING_RAY_FINE_TUNING_DIR / "artifacts/job_pod.log")) as f:
-        for line in f.readlines():
-            if not line.startswith("---"):
-                continue
-            ray_metrics.summary.time = float(line.strip().split("::: ")[-1].split()[0])
-            break
+
+        if ...:
```

Comment:
do we want to keep the elipsis or evaluate a condition to parse the appropriate test flavor results?